### PR TITLE
Issue #6701: create osx related builds in travis.sh

### PR DESCRIPTION
**Pull request by ahmadshabib**
Issue #6701

### CODE REVIEW

**Comment by rnveach** [comment]
https://travis-ci.org/checkstyle/checkstyle/jobs/531251403
> ./.ci/travis/travis.sh: line 129: conditional binary operator expected

File: .travis.yml [hunk 1]
```diff
@@ -156,31 +156,23 @@ matrix:
...
 
     # MacOS JDK11 verify (till cache is not working, we can not do verify)
     - os: osx
       osx_image: xcode10.1
       env:
         - DESC="MacOS JDK11 verify, site, assembly
-        - CMD1="mvn -e package -Dlinkcheck.skip=true"
-        - CMD2="mvn -e package -Passembly"
-        - CMD="export JAVA_HOME=$(/usr/libexec/java_home) && $CMD1 && $CMD2"
+        - CMD="./.ci/travis/travis.sh travis-osx"
```
Comment (romani) [inline .travis.yml]: Please call it as:
./.ci/travis/travis.sh osx-package
./.ci/travis/travis.sh osx-assembly

**Comment by ahmadshabib** [comment]
> https://travis-ci.org/checkstyle/checkstyle/jobs/531251403
> 
> > ./.ci/travis/travis.sh: line 129: conditional binary operator expected

any idea why this is failing since i didn't change that specific line ?

File: .travis.yml (hunk 1 above)
Comment (ahmadshabib) [reply .travis.yml]: Done

**Pull request merged**